docs/building.md

```diff
@@ -0,0 +1,50 @@
+Building the extension
+======================
+
+This document describes how to build the browser extension and load it into your
+browser.
+
+Installing the dependencies
+---------------------------
+
+You should have [Node v4.x][node4] installed.
+
+Before continuing, install the development dependencies by running the following
+at the root of the repository:
+
+    $ npm install
+
+[node4]: https://nodejs.org/en/download/
+
+Building the extension
+----------------------
+
+The extension build is configured by a JSON settings file, some examples of
+which are supplied in the `settings/` directory. To build the extension, you
+simply run `make`. To build the extension in its default configuration (a Chrome
+development build), run:
+
+    $ make
+
+To build the extension from an alternate settings file, simply specify the
+`SETTINGS_FILE` variable when running `make`:
+
+    $ make SETTINGS_FILE=settings/chrome-prod.json
+
+This will build a production extension.
```

Comment:
I think we should be a little clearer here about what a "production extension" means, since it is the easiest way to build a customized version of the client.  Perhaps: "This will build a Chrome extension that uses the main Hypothesis web service."